checkov/terraform/checks/resource/gcp/DataprocPrivateCluster.py

```diff
@@ -0,0 +1,40 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from typing import List
+
+
+class DataprocPrivateCluster(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that Dataproc clusters are not anonymously or publicly accessible"
+        id = "CKV_GCP_95"
+        supported_resources = ['google_dataproc_cluster_iam_member', 'google_dataproc_cluster_iam_binding']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        public_principals = (
+            "allUsers",
+            "allAuthenticatedUsers"
+            )
+        # Depending on the terraform resource type -
+        # The member config is either a list or single principal
+        if self.entity_type == "google_dataproc_cluster_iam_member":
+            # conf.get returns as a list
+            # so we create a string for comparison
+            member = ''.join(conf.get("member"))
+            if member in public_principals:
+                return CheckResult.FAILED
+            else:
+                return CheckResult.PASSED
```

Comment:
```suggestion
            member = conf.get("member")[0]
            if member in public_principals:
                return CheckResult.FAILED
            else:
                return CheckResult.PASSED
```
similar to what you wrote a few lines lower